service/storage_service.cc

```diff
@@ -3554,5 +3555,108 @@ db::schema_features storage_service::cluster_schema_features() const {
     return f;
 }
 
-} // namespace service
+future<> storage_service::start_redis_transport(distributed<redis::query_processor>& qp) {
```

Comment:
Can this code please moved to one of the source files in redis/? Maybe redis/server.cc or something?
I'd really like to see most of the redis-related code in redis/, most of the DynamoDB-specific code in alternator/, etc. storage_service.cc is already too huge.

Reply:
Yes, of course.